File: android/app/src/main/java/com/xirigo/ecommerce/feature/home/domain/model/HomeProduct.kt
package com.xirigo.ecommerce.feature.home.domain.model

data class HomeProduct(
    val id: String,
    val title: String,
    val imageUrl: String?,
    val price: String,
    val currencyCode: String,
    val originalPrice: String?,
    val vendor: String,
    val rating: Float?,
    val reviewCount: Int?,
    val isNew: Boolean,
)

File: ios/XiriGoEcommerce/Feature/Home/Domain/Model/HomeProduct.swift
import Foundation

// MARK: - HomeProduct

/// A product displayed in Popular Products or New Arrivals sections on the home screen.
struct HomeProduct: Identifiable, Equatable, Sendable {
    let id: String
    let title: String
    let imageUrl: String?
    let price: String
    let currencyCode: String
    let originalPrice: String?
    let vendor: String
    let rating: Double?
    let reviewCount: Int?
    let isNew: Bool
}
